js/src/utils/getHighestBudget.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * If a merchant selects more than one country, the budget recommendation
+ * takes the highest country out from the selected countries.
+ *
+ * For example, a merchant selected Brunei (20 USD) and Croatia (15 USD),
+ * then the budget recommendation should be (20 USD).
+ */
+export default function getHighestBudget( recommendations ) {
```

Comment:
@dsawardekar We have `getHighestBudget` [defined here as well](js/src/components/paid-ads/budget-section/budget-recommendation/index.js). Can you remove the function from there and instead import it from `js/src/utils/getHighestBudget.js`?

Reply:
Thanks @dsawardekar